package.json

```diff
@@ -1,54 +1,48 @@
 {
   "name": "orbit-db-test-utils",
-  "version": "0.8.4",
+  "version": "0.8.5",
   "description": "",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/orbitdb/orbit-db-test-utils"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "TEST=all nyc mocha"
   },
   "bin": {
     "redis-server": "./bin/redis-server"
   },
+  "standard": {
+    "env": "mocha"
+  },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "babel-loader": "^8.0.6",
     "datastore-level": "^0.14.1",
-    "fruitdown": "^1.0.2",
-    "go-ipfs-dep": "~0.4.20",
-    "ipfs": "~0.40.0",
-    "ipfs-repo": "^0.30.1",
-    "ipfsd-ctl": "^0.47.4",
+    "go-ipfs-dep": "~0.4.23-3",
+    "ipfs": "~0.43.0",
+    "ipfs-repo": "^1.0.1",
+    "ipfsd-ctl": "^3.1.0",
     "is-node": "^1.0.2",
```

Comment:
Not sure if we need `is-node` anymore due to the sniffing technique used in the `./implementations/index/js`